package.json

```diff
@@ -63,9 +63,8 @@
     "pino-filter": "^1.0.0",
     "pino-http": "^5.3.0",
     "pino-pretty": "^4.3.0",
-    "sass": "^1.29.0",
-    "react-i18next": "^11.7.3",
     "sass": "^1.26.10",
```

Comment:
When you merge `package.json` please ensure only one `sass` entry and version `1.29.0`

Reply:
Thought I fixed that thanks for the catch.